backend/app/assets/stylesheets/spree/backend/sections/_style_guide.scss

```diff
@@ -106,6 +106,15 @@
       border: 1px solid darken($color-7, 10%);
     }
   }
+
+  table {
```

Comment:
The table needs a class otherwise all tables in the style guide will match this.